perfkitbenchmarker/sample.py

```diff
@@ -57,11 +58,18 @@ class Sample(collections.namedtuple('Sample', _SAMPLE_FIELDS)):
     value: float. Result for 'metric'.
     unit: string. Units for 'value'.
     metadata: dict. Additional metadata to include with the sample.
+    timestamp: float. Unix timestamp.
   """
 
-  def __new__(cls, metric, value, unit, metadata=None, **kwargs):
+  def __new__(cls, metric, value, unit, metadata=None, timestamp=None,
+              **kwargs):
+    if timestamp is None:
+      timestamp = time.time()
```

Comment:
Making this change in the sample class will alter current behavior (samples from the same benchmark could have fairly different timestamps, whereas currently they are clustered around a single point in time).  I'm not sure that's a bad thing, but maybe worth a discussion?